### DISCUSSION TITLE
[教程] 可视化学习Git利器Learn Git Branching

### DISCUSSION THREAD

**Discussion by tyn1998** [X-lab2017/open-wonderland#315] [1️⃣ 新鲜事]
Git是你在软件世界行走的腿脚，会用Git和熟练掌握Git是两回事情。熟练掌握Git可以大大降低试错成本和提升并行处理任务的能力。理解Git的本质对熟练掌握Git有极大的促进作用，下面这个网站以可视化形式提供了一系列的Git分支操作教程，我认为做完这些教程是能对Git有更深刻的了解的。

https://learngitbranching.js.org/

<img width="1369" alt="image" src="https://github.com/X-lab2017/open-wonderland/assets/32434520/b55c1d06-9380-452d-b987-3963a05c294d">

强烈推荐新同学 @X-lab2017/class23 把整套教程走完~

**Comment by HalloMelon** [comment]
so so so so thankful！！！！

**Comment by Tenth-crew** [comment]
Greatttttttttttttttt!

**Comment by wangyantong2000** [comment]
OK！

**Comment by andyhuang18** [comment]
回想起去年也曾犯许多小错误，多尝试，有问题多问！
